### PR TITLE
Use taraxa cli

### PR DESCRIPTION
**Pull request by sfrique**
Use taraxa-cli as config generator and remove the faucet stuff, this will be handled by the website manually for each user.

The taraxa-cli doesn't exist in docker hub yet. We need to wait to be published.

### CODE REVIEW

File: scripts/ubuntu-install-and-run-node.sh [hunk 1]
```diff
@@ -87,54 +24,24 @@ sudo add-apt-repository \
 sudo apt-get update
 sudo apt-get install -y docker-ce docker-ce-cli containerd.io git jq
```
Comment (justinsnapp) [inline scripts/ubuntu-install-and-run-node.sh:25]: Testing this in Ubuntu 20 04 seemed to give me trouble.    I had to use this info:  https://unix.stackexchange.com/questions/363048/unable-to-locate-package-docker-ce-on-a-64bit-ubuntu

Comment (sfrique) [reply scripts/ubuntu-install-and-run-node.sh:25]: I tested on ubuntu 20.04 as well, and it worked.

The info you posted, see if there isn't the few lines above (add the key and docker repository)

After this we also need to change the other scripts touse ubntu 20.04, as of now it is on 18.04
but script should work on both.

File: scripts/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        ports:
+            - "10002:10002"
+            - "10002:10002/udp"
+            - "7777:7777"
+            - "8777:8777"
+        expose:
+            - "10002"
+            - "7777"
+            - "8777"
+        entrypoint:
```
Comment (sfrique) [inline scripts/docker-compose.yml:18]: No need to use this anymore right?

File: scripts/ubuntu-install-and-run-node.sh [hunk 3]
```diff
@@ -87,54 +23,17 @@ sudo add-apt-repository \
...
+sudo docker run -it --name taraxa \
+        -v ${TARAXA_NODE_PATH}:/taraxa \
+        -e DEBUG=1 \
+        -p 10002:10002 \
+        -p 7777:7777 \
+        -p 8777:8777 \
+        -p 10002:10002/udp \
+        --restart always \
+        --entrypoint /usr/bin/sh \
+        $TARAXA_NODE_DOCKER_IMAGE -c "taraxa config -n testnet -d /var/taraxa && ./docker-entrypoint.sh --conf_taraxa /var/taraxa/conf/testnet.json"
```
Comment (sfrique) [inline scripts/ubuntu-install-and-run-node.sh:39]: the path `/var/taraxa` is different from the one mounted. So either we change here or there to actually have the data at the host
(or create a docker volume as we do at docker-compose)

File: scripts/ubuntu-install-and-run-node.sh [hunk 4]
```diff
@@ -87,54 +23,17 @@ sudo add-apt-repository \
...
+# #Generate config
+sudo docker run -it --name taraxa \
+        -v ${TARAXA_NODE_PATH}:/taraxa \
+        -e DEBUG=1 \
+        -p 10002:10002 \
+        -p 7777:7777 \
+        -p 8777:8777 \
+        -p 10002:10002/udp \
+        --restart always \
+        --entrypoint /usr/bin/sh \
```
Comment (sfrique) [inline scripts/ubuntu-install-and-run-node.sh:38]: No need to use this anymore right? [2]

File: scripts/ubuntu-install-and-run-node.sh (hunk 3 above)
Comment (dmidnight) [reply scripts/ubuntu-install-and-run-node.sh:39]: fixed path

File: scripts/docker-compose.yml (hunk 2 above)
Comment (dmidnight) [reply scripts/docker-compose.yml:18]: as of next image version, correct

File: scripts/ubuntu-install-and-run-node.sh (hunk 3 above)
Comment (sfrique) [reply scripts/ubuntu-install-and-run-node.sh:39]: The actual host path doesn't matter, what needs to be fixed is where it mounts at the container:
`-v ${TARAXA_NODE_PATH}:/var/taraxa`

So whatever path in the var, will container `/var/taraxa`  content, (which you choose to use as `-d` argument for the configuration. (i think the data also goes there right?

File: scripts/docker-compose.yml (hunk 2 above)
Comment (sfrique) [reply scripts/docker-compose.yml:18]: oh, I thought that was pushed already.

File: scripts/ubuntu-install-and-run-node.sh [hunk 5]
```diff
@@ -87,54 +23,17 @@ sudo add-apt-repository \
...
-export RECV_ADDRESS=${MY_ADDRESS}
-export RPC_PORT=${TARAXA_LOCAL_RPC_PORT}
-export NODE=${TARAXA_LOCAL_NODE_ADDRESS}
-#python3 /opt/taraxa-ops/scripts/send_coins_to_self.py
-#nohup python3 /opt/taraxa-ops/scripts/send_coins_to_self.py > send_coins_to_self.log &
-
-# Add scripts folder to PATH for root user
-echo 'export PATH=$PATH:/opt/taraxa-ops/scripts' >> /root/.profile
+# #Generate config
+sudo docker run -it --name taraxa \
```
Comment (sfrique) [inline scripts/ubuntu-install-and-run-node.sh:30]: We should add as `-d` and remove `-it`  this script will be used to automatic provision the node.

**Pull request merged**